### ISSUE TITLE
NaN loss when training from scratch

### ISSUE THREAD

**Issue by saeedizadi** [imatge-upc/saliency-salgan-2017#9]
Hi,
I tried to train your model on my own dataset consisting of (RGB image, Binary mask) pairs which both are images (and not Mat file). However, after several epochs, I get NaN train loss. what's the problem? I modified your 01-preprocess_data code to use binary images as the ground truth. does it interfere the training?

**Comment by junting** [comment]
Hello @saeedizadi ,

I do not think that you are getting NaN train loss because the use of Binary mask. Please try to use a smaller learning rate, maybe the defaults learning rate is making your training diverging.

**Comment by saeedizadi** [comment]
Yes, you are right. fixed
